packager/app/packager_main.cc

```diff
@@ -296,9 +331,18 @@ bool ParseProtectionSystems(const std::string& protection_systems_str,
       {"widevine", ProtectionSystem::kWidevine},
   };
 
-  for (const std::string& protection_system :
-       base::SplitString(base::ToLowerASCII(protection_systems_str), ",",
-                         base::TRIM_WHITESPACE, base::SPLIT_WANT_NONEMPTY)) {
+  auto tokens = absl::StrSplit(absl::AsciiStrToLower(protection_systems_str),
+                               ',', absl::SkipEmpty());
+  std::vector<std::string> protection_systems_vec;
+  for (const absl::string_view& token : tokens) {
+    std::string trimmed = std::string(token);
+    absl::StripAsciiWhitespace(&trimmed);
+    if (!trimmed.empty()) {
+      protection_systems_vec.push_back(trimmed);
+    }
+  }
```

Comment:
I feel like I've seen this pattern a few times now.  Maybe this could be factored out into a utility function somewhere?